source/wp-content/themes/wporg-developer/scss/main.scss

```diff
@@ -2463,35 +2463,20 @@ body.responsive-show {
 
 		.code-reference & {
 			margin-left: 0;
-			left: calc( #{ $devhub-wrap-content-width } + ( ( 100vw - #{ $devhub-wrap-content-width } ) / 2 ) );
+			transform: translateX( calc( #{ $devhub-wrap-content-width } - #{ $devhub-wrap-content-padding-horizontal } ) );
 			top: $devhub-wrap-toc-position-top;
-			max-height: calc(90vh - #{ $devhub-wrap-toc-position-top });
+			max-height: calc( 90vh - #{ $devhub-wrap-toc-position-top } );
 		}
 
 		// Pages with sidebar
 		.widget-area + & {
-			left: calc( #{ $single-handbook-content-width } + ( ( 100vw - #{ $single-handbook-content-width } ) / 2 ) );
+			transform: translateX( calc( ( #{ $single-handbook-content-width } * #{ $single-handbook-content-width-primary-proportion } ) - #{ $single-handbook-content-padding-horizontal } ) );
 			top: $single-handbook-toc-position-top;
 			max-height: calc(90vh - #{ $single-handbook-toc-position-top });
 		}
 	}
 }
 
-@media (min-width: 877px) {
-	// Pages which are not code reference and without a sidebar
```

Comment:
This was required for local dev, but in production all pages have a sidebar on the left so this code is redundant